source/expressions.tex

```diff
@@ -281,31 +281,21 @@
 \end{note}
 
 \pnum
-If a program attempts to access the stored value of an object through a glvalue
-of other than one of the following types the behavior is
-undefined:\footnote{The intent of this list is to specify those circumstances in which an
-object may or may not be aliased.}
+If a program attempts to access the stored value of an object of type \tcode{T}
+through a glvalue whose dynamic type is not one of the following types,
+the behavior is undefined:\footnote{The intent of this list is
+to specify those circumstances in which an object may or may not be aliased.}
 \begin{itemize}
-\item the dynamic type of the object,
+\item \cv{} \tcode{T}
 
-\item a cv-qualified version of the dynamic type of the object,
+\item a type similar (as defined in~\ref{conv.qual}) to \tcode{T},
 
-\item a type similar (as defined in~\ref{conv.qual}) to the dynamic type
-of the object,
-
-\item a type that is the signed or unsigned type corresponding to the
-dynamic type of the object,
-
-\item a type that is the signed or unsigned type corresponding to a
-cv-qualified version of the dynamic type of the object,
+\item a type that is the signed or unsigned type corresponding to \cv{} \tcode{T},
 
 \item an aggregate or union type that includes one of the aforementioned types among its
 elements or non-static data members (including, recursively, an element or non-static data member of a
 subaggregate or contained union),
```

Comment:
This is useless in C++, except for the copy of a union. Oh, covered by the final bullet, which covers object representations = arrays of unsigned char.